mandatory/3-function-output.js

```diff
@@ -1,14 +1,18 @@
 // Add comments to explain what this function does. You're meant to use Google!
 function getRandomNumber() {
   return Math.random() * 10;
+  // The math function returns a number between 0-9.
 }
 
 // Add comments to explain what this function does. You're meant to use Google!
 function combine2Words(word1, word2) {
   return word1.concat(word2);
+  // The .concat method connects two or more strings.
 }
 
-function concatenate(firstWord, secondWord, thirdWord) {
+function concatenate(Hello, toxic, world) {
```

Comment:
We use camelCase for JS parameters. So `Hello` shall be reserved for JS class names that you will learn in next class. You have done a great job!  Be Happy! Cheers : >